lib/internal/test_runner/reporter/dot.js

```diff
@@ -1,24 +1,30 @@
 'use strict';
+
+const { green, red } = require('internal/util/colors');
 const { MathMax } = primordials;
 
 module.exports = async function* dot(source) {
   let count = 0;
   let columns = getLineLength();
+
   for await (const { type } of source) {
     if (type === 'test:pass') {
-      yield '.';
+      // Made green dot for pass
+      yield green('.');
     }
+
     if (type === 'test:fail') {
-      yield 'X';
+      // Made red dot for fail
+      yield red('X');
     }
-    if ((type === 'test:fail' || type === 'test:pass') && ++count === columns) {
-      yield '\n';
 
-      // Getting again in case the terminal was resized.
+    if ((type === 'test:fail' || type === 'test:pass') && ++count === columns) {
+      // Getting the line length again in case the terminal was resized.
```

Comment:
```suggestion
	  yield '\n';
      // Getting the line length again in case the terminal was resized.
```
please revert this removal